conf/app.routes

```diff
@@ -1,3 +1,5 @@
 # microservice specific routes
 
-POST        /claim         uk.gov.hmrc.cdsreimbursementclaim.controllers.SubmitClaimController.claim
\ No newline at end of file
+POST        /claim                   uk.gov.hmrc.cdsreimbursementclaim.controllers.SubmitClaimController.claim
+
+GET         /declaration/:id             uk.gov.hmrc.cdsreimbursementclaim.controllers.GetDeclarationController.declaration(id:String)
```

Comment:
can you create a path bindable here so it only accepts strings which match the structure of a declarant id (ie an MRN number)